astro.config.mjs

```diff
@@ -3,6 +3,15 @@ export default /** @type {import('astro').AstroUserConfig} */ ({
 	buildOptions: {
 		site: 'https://docs.astro.build/',
 	},
+	markdownOptions: {
```

Comment:
this is what broke the build! Grab the new version (commented out) from the other PR.